src/main/java/com/minecraftcorp/lift/bukkit/listener/PlayerListener.java

```diff
@@ -164,6 +164,35 @@ public void onPlayerMove(PlayerMoveEvent event) {
 		player.setVelocity(pushBack);
 	}
 
+	@EventHandler
+	public void onPlayerChangedWorldEvent(PlayerChangedWorldEvent event) {
+		Player player = event.getPlayer();
+		Location playerLocation = player.getLocation();
```

Comment:
If we need the location for teleport, can we move this line down right before we need it?

Reply:
Probably not possible. At the beginning of the implementation, I thought it was necessary to get the location information of the destination as soon as the handler was called, so we wrote it as far up the line as possible.

However, I have just now discovered that teleportation is not necessary, so I have removed this code.